netbout-client/pom.xml

```diff
@@ -178,11 +178,6 @@
                     <plugin>
                         <groupId>com.qulice</groupId>
                         <artifactId>qulice-maven-plugin</artifactId>
-                        <configuration>
-                            <excludes>
-                                <exclude>findbugs:.*</exclude>
```

Comment:
@dmzaytsev thanks, it's clear. But what about puzzle creation? see below question